server/config/config.go

```diff
@@ -18,11 +18,13 @@ import (
 )
 
 const (
-	defaultGrpcHandleTimeoutMs int64 = 60 * 1000
-	defaultEtcdStartTimeoutMs  int64 = 60 * 1000
-	defaultCallTimeoutMs             = 5 * 1000
-	defaultMaxTxnOps                 = 128
-	defaultEtcdLeaseTTLSec           = 10
+	defaultGrpcHandleTimeoutMs     int64 = 60 * 1000
+	defaultGrpcOperationBurstCount int   = 100 * 1000
```

Comment:
Maybe we need some comment to explain what the config and the default value does.